cellfinder/napari/train/train.py

```diff
@@ -18,24 +19,71 @@
 )
 
 
-@thread_worker
-def run_training(
-    training_data_inputs: TrainingDataInputs,
-    optional_network_inputs: OptionalNetworkInputs,
-    optional_training_inputs: OptionalTrainingInputs,
-    misc_training_inputs: MiscTrainingInputs,
-):
-    print("Running training")
-    train_yaml(
-        **training_data_inputs.as_core_arguments(),
-        **optional_network_inputs.as_core_arguments(),
-        **optional_training_inputs.as_core_arguments(),
-        **misc_training_inputs.as_core_arguments(),
-    )
-    print("Finished!")
+class MyWorkerSignals(WorkerBaseSignals):
```

Comment:
```suggestion
class TrainingWorkerSignals(WorkerBaseSignals):
```

is a lot better name for this, I think